presto-sqlserver/src/test/java/io/prestosql/plugin/sqlserver/TestSqlServerIntegrationSmokeTest.java

```diff
@@ -417,6 +417,30 @@ public void testShowCreateForPartitionedTablesWithDataCompression()
                         ")");
     }
 
+    @Test
+    public void testShowCreateForIndexedAndCompressedTable()
```

Comment:
Please add comment that `SHOW CREATE TABLE` does not support indices and table compression yet.